src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -1296,11 +1296,25 @@ private static bool IsPackageReferenceAutoReferenced(PackageReference package)
             return nuGetProjectActions;
         }
 
+        /// <summary>
+        /// The package dependency info for the given project. The project needs to be packages.config, otherwise returns an empty list.
```

Comment:
nope, it won't. It would've thrown. I also don't see a reason to make it work retroactively yet. So this just makes it more explicit. It's only used in the migrator right now.